main/zoe/guide/contracts/atomic-swap.md

```diff
@@ -123,8 +104,8 @@ moolaPayment is empty, and the simoleanPayment has a balance of 7.
 The same is true for Bob, but for his specific payout.
```

Comment:
>> Now that Bob has made his offer, Alice's `payout` resolves to an array	

I'd say 

"Now that Bob has made his offer, the contract executes and Alice's `payout` resolves to an array	" or something similar so people don't get the idea that Bob's offer normally gets funnelled straight through to Alice. It's just atomic-swap.

Reply:
Yeah good point. We need to replace the use of "array" in that sentence too. It's a record with keyword keys now.

Reply:
@katelynsills 
Further into the sentence it provides an array. Would we update that to:
`{ Asset: moola.make(0), Price: simoleans.make(7) }` or
`{ Asset: 0, Price: 7 }` or something else?

Reply:
Let's update it to `{ Asset: moola(0), Price: simoleans(7) }`. We've been using the names as a shorthand, assuming that somewhere earlier in the code we have done:

```
const moola = moolaAmountMath.make;
const simoleans = simoleanAmountMath.make;
``` 

In fact, let's add the above snippet to the example if we can to make it very clear what we are doing.